README.md

```
@@ -147,18 +147,38 @@ Experiment:
-```bash
-uvicorn src.api.main:app --reload
-```
-
-## Frontend
-
-```bash
-streamlit run frontend/streamlit_app.py
-```
-
----
+  # Запуск проекта
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)